### PR TITLE
message pt through marketplace

### PR DESCRIPTION
**Pull request by euniceyoh**
if user is registered with sendbird, user can initiate and send a message through marketplace, which should show up when they go to the chat tab. 
The PT and Client must be a user already (which is done when a user logs in for the first time, and renders the Chat app).

I've also disabled clients being able to send anyone a message now, it should only be possible through marketplace.

### CODE REVIEW

File: src/components/Chat/Chat.jsx [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+        userId={userId} 
+        nickname={props.user.publicName}
+        profileUrl={props.user.pictureUrl}
+        userListQuery={[]}
+      >
+        <div className='sendbird-app__wrap'>
+          {/* <div className = 'sendbird-channel-header sendbird-channel-header__right-icon'> */}
+          <ChannelList 
+          onChannelSelect={(channel) => {
+                if(channel != undefined) {
```
Comment (charlotte-zhuang) [inline src/components/Chat/Chat.jsx]:
```suggestion
                if(channel) {
```
I think the code should work either way.

File: src/components/Chat/ChatButton.jsx [hunk 2]
```diff
@@ -0,0 +1,89 @@
...
+        console.log(message)
+        console.log(props.pt)
+
+        let channelParams = new sdk.GroupChannelParams()
+        channelParams.isPublic = false
+        channelParams.isEphemeral = false
+        channelParams.isDistinct = true 
+      
+        const ptId = props.pt.id
+        var userIds = [ptId]
```
Comment (charlotte-zhuang) [inline src/components/Chat/ChatButton.jsx]:
```suggestion
        const userIds = [ptId]
```
Does this need to be a var?

**Review by charlotte-zhuang — APPROVED** [review]
It looks like it's working for me.

**Comment by christinang3030** [comment]
So I get the popup in Marketplace that says "send initial message..." but when I go to the Chat tab, the (No name) shows up in the top left, and below that I think is just my client id. 
image

File: src/components/Chat/ChatButton.jsx (hunk 2 above)
Comment (euniceyoh) [reply src/components/Chat/ChatButton.jsx]: nope

**Comment by euniceyoh** [comment]
@christinang3030 did you update the APP_ID?

**Review by isabellapiz — APPROVED** [review]
worked for me, nice job!

**Review by christinang3030 — APPROVED** [review]
Works for me now too! I had to update the app ID.

**Pull request merged**